test/vmtester/mock_context.cpp

```diff
@@ -0,0 +1,136 @@
+#include <evmc/evmc.h>
+#include <evmc/helpers.h>
+
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+struct evmc_uint256be balance(struct evmc_context* context, const struct evmc_address* address)
```

Comment:
You don't need `struct` prefix in C++.

Reply:
This is a copy from `examples/capi.c`. Should we keep it as C (with a `.c` extension) or go full in on C++?